app/javascript/packs/homePageFeed.jsx

```diff
@@ -81,6 +81,7 @@ export const renderFeed = (timeFrame) => {
         };
 
         const [featuredStory, ...subStories] = feedItems;
+        const feedStyle = JSON.parse(document.body.dataset.user).feed_style;
```

Comment:
Similar to Molly's comment at https://github.com/thepracticaldev/dev.to/pull/8721/files#r441585424, it feels odd to be pulling a site configuration option from user information